package.json

```diff
@@ -30,64 +30,66 @@
   },
   "private": true,
   "dependencies": {
-    "@alfresco/adf-content-services": "7.0.0-alpha.9-13674229115",
-    "@alfresco/adf-core": "7.0.0-alpha.9-13674229115",
-    "@alfresco/adf-extensions": "7.0.0-alpha.9-13674229115",
-    "@alfresco/eslint-plugin-eslint-angular": "7.0.0-alpha.9-13674229115",
-    "@alfresco/js-api": "8.0.0-alpha.9-13674229115",
-    "@angular/animations": "17.1.3",
-    "@angular/cdk": "17.1.2",
-    "@angular/common": "17.1.3",
-    "@angular/compiler": "17.1.3",
-    "@angular/core": "17.1.3",
-    "@angular/forms": "17.1.3",
-    "@angular/material": "17.1.2",
-    "@angular/material-date-fns-adapter": "17.1.2",
-    "@angular/platform-browser": "17.1.3",
-    "@angular/platform-browser-dynamic": "17.1.3",
-    "@angular/router": "17.1.3",
+    "@alfresco/adf-content-services": "8.1.0-13905331726",
+    "@alfresco/adf-core": "8.1.0-13905331726",
+    "@alfresco/adf-extensions": "8.1.0-13905331726",
+    "@alfresco/eslint-plugin-eslint-angular": "8.1.0-13905331726",
+    "@alfresco/js-api": "9.1.0-13905331726",
+    "@angular/animations": "18.2.9",
+    "@angular/cdk": "18.2.9",
+    "@angular/common": "18.2.9",
+    "@angular/compiler": "18.2.9",
+    "@angular/core": "18.2.9",
+    "@angular/forms": "18.2.9",
+    "@angular/material": "18.2.9",
+    "@angular/material-date-fns-adapter": "18.2.9",
+    "@angular/platform-browser": "18.2.9",
+    "@angular/platform-browser-dynamic": "18.2.9",
+    "@angular/router": "18.2.9",
     "@fontsource/open-sans": "^5.1.0",
-    "@mat-datetimepicker/core": "13.0.2",
-    "@ngrx/effects": "17.0.1",
-    "@ngrx/router-store": "17.0.1",
-    "@ngrx/store": "17.0.1",
-    "@ngrx/store-devtools": "17.0.1",
+    "@mat-datetimepicker/core": "14.0.0",
+    "@ngrx/effects": "18.1.1",
+    "@ngrx/operators": "^18.0.0",
```

Comment:
Rest of ngrx versions has not ^ so should we remove ^ here as well to have the same?